project_event/tests/common.py

```diff
@@ -178,3 +178,41 @@ def setUp(self):
             'room_id': self.room_1.id,
             'notes': 'Some Task Notes ...',
         })
+        self.task_3 = self.Tasks.create({
+            'name': 'Sample task3',
+            'activity_task_type': 'activity',
+            'partner_id': self.project_1.partner_id.id,
+            'room_id': self.room_1.id,
+            'date_start': fields.Datetime.to_string(datetime.today()),
+            'date_end': fields.Datetime.to_string(datetime.today() +
+                                                  timedelta(hours=4)),
+            'real_date_start': fields.Datetime.to_string(datetime.today()),
+            'real_date_end': fields.Datetime.to_string(datetime.today()),
+            'total_time': " ",
+        })
+        self.task_4 = self.Tasks.create({
+            'name': 'Sample task4',
+            'activity_task_type': 'activity',
+            'partner_id': self.project_1.partner_id.id,
+            'room_id': self.room_1.id,
+            'date_start': fields.Datetime.to_string(datetime.today()),
+            'date_end': fields.Datetime.to_string(datetime.today() +
+                                                  timedelta(hours=4)),
+            'real_date_start': fields.Datetime.to_string(datetime.today()),
+            'real_date_end': fields.Datetime.to_string(datetime.today() +
+                                                       timedelta(hours=2)),
+            'total_time': " ",
+         })
+        self.activity_3 = self.Tasks.create({
+            'name': 'Test Activity 3',
+            'activity_task_type': 'activity',
+            'project_id': self.project_1.id,
+            'responsible_id': self.project_1.responsible_id.id,
+            'partner_id': self.project_1.partner_id.id,
+            'category_id': self.category_1.id,
+            'room_id': self.room_1.id,
+            'spectators': '-',
+            'date_start': fields.Datetime.to_string(datetime.today()),
+            'date_end': fields.Datetime.to_string(datetime.today() +
+                                                  timedelta(hours=4)),
+        })
```

Comment:
@eilst  . You can write a test case where the time difference is only on minutes

Reply:
Hey hey, totally agree...but, this is your code, we merge it from
[ADD] Add section Time spent tasks (#352)this from https://github.com/savoirfairelinux/project-addons/pull/378/commits/382a162853b16f33813b8d3f338f47a7682fc3f8